src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -60,8 +60,18 @@ function DomainSlider(props: Props): ReactElement {
         isAutoMin={isAutoMin}
         isAutoMax={isAutoMax}
         disabled={disabled}
-        onChange={setSliderDomain}
-        onAfterChange={onCustomDomainChange}
+        onChange={(newValue, hasMinChanged, hasMaxChanged) => {
+          setSliderDomain([
+            hasMinChanged ? newValue[0] : sliderDomain[0],
+            hasMaxChanged ? newValue[1] : sliderDomain[1],
+          ]);
+        }}
```

Comment:
You could keep `onChange={setSliderDomain}` here and put the ternaries inside `ScaledSlider`.